docs/guides/Common-Navigation-Spec.md

```diff
@@ -56,8 +56,8 @@ A navigation-aware component that hosts other navigation-aware components. Most
 The navigation prop should be provided to components who need access to navigation. If provided, it must follow this interface:
 
 ```javascript
-type BackAction = {type: 'Back'};
-type URIAction = {type: 'URI', uri: string};
+type BackAction = {type: 'navigation/BACK'};
```

Comment:
Let's make it `Navigation/Back` ? Again, all lowercase prefix and all uppercase name looks weird